agent/action/get_state.go

```diff
@@ -68,6 +68,16 @@ func (a GetStateAction) Run(filters ...string) (GetStateV1ApplySpec, error) {
 			return GetStateV1ApplySpec{}, bosherr.WrapError(err, "Building full vitals")
 		}
 		vitalsReference = &vitals
+		services := a.jobSupervisor.ServiceStatus()
+		processStatistics := []map[string]interface{}{}
+		for _, service := range services {
+			ps := map[string]interface{}{
+				"name":  service.Name,
+				"state": service.Status,
+			}
+			processStatistics = append(processStatistics, ps)
+		}
+		vitals.Process = processStatistics
```

Comment:
instead of adding more things to vitals lets add `Processes` (array) to `GetStateV1ApplySpec`